row.go

```diff
@@ -1,9 +1,16 @@
 package xlsx
 
 type Row struct {
-	Cells  []*Cell
-	Hidden bool
-	Sheet  *Sheet
+	Cells    []*Cell
+	Hidden   bool
+	Sheet    *Sheet
+	Height   float64
+	IsCustom bool
```

Comment:
`IsCustom` is about `Height`, but the name doesn't make that clear. Maybe `CustomHeight` as done in `xlsxRow`? I like the `Is` prefix (maybe even make it `is`, since it doesn't have to be shared outside the package.